tests/sentry/integrations/msteams/test_utils.py

```diff
@@ -28,3 +37,103 @@
     "type": "conversationUpdate",
     "id": "f:8e005ef8-f848-156f-55b1-0a5bb3207225",
 }
+
+
+class GetChannelIdTest(TestCase):
+    def setUp(self):
+        self.resp = responses.mock
+        self.resp.__enter__()
```

Comment:
@KKaempen I don't think you should need `self.resp`. Guessing the problem is that you're not using the decorator `@responses.activate` on the tests.